spec/lib/import_csv_to_big_query_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'import_csv_to_big_query'
+require 'rails_helper'
```

Comment:
Is this required, looks like you've completely decoupled from Rails at this point.

Reply:
The tests fail without it, I think it's partly because the `underscore` method is a rails method